src/data/KeyHandler.java

```diff
@@ -0,0 +1,2 @@
+package data;public class KeyHandler {
```

Comment:
These things should not be on the same line.

Reply:
If my memory serves me correctly, you were the one who implemented this :)

Reply:
I will fix this once we transition to real-world practice